src/utils/NumberUtils.h

```diff
@@ -19,7 +19,7 @@
 #include <system_error>
 #include <type_traits>
 
-namespace NumberUtils
+namespace OCIO_NAMESPACE
```

Comment:
I did not say to remove the `NumberUtils` namespace but rather to encapsulate it in the `OCIO_NAMSPACE` one i.e. two namespaces. That's to have code like `NumberUtils::from_chars()` (like `Platform::Getenv()` for example) to avoid potential confusion with `std::from_chars`.